src/foam/box/Message.js

```diff
@@ -59,6 +67,26 @@ foam.CLASS({
         Message replyMessage = new Message();
         replyMessage.setObject(reply);
 
+        replyBox.send(replyMessage);
+      `
+    },
+    {
+      name: 'replyWithCapabilityRequired',
```

Comment:
This doesn't strike me as desirable at first glance, although I'm not thoroughly convinced it's not. I'd consider this behaviour just a specific case of `replyWithException`, and would argue against changing the interface of message because of that. I think it would make more sense to inline this code above rather than extracting it to a method.

Reply:
That's what I tried to do at first, but it significantly complicates things because I want to use a different class in place of RemoteException. I didn't want to spend too much time on it because I'm hoping to remove `CapabilityRequiredRemoteException` alltogether when I add modelled exceptions.